File: 2000-2999/OpenJudge2723.cpp
#include <cstdio>
#include <iostream>
#include <cstring>
#include <string>

using namespace std ;

// Number of days per month
int daysPerMonth[] = {0, 31, 28, 31, 30, 31, 30, 
				31, 31, 30, 31, 30} ;


int main(int argc, char *argv[]) 
{
	int whatday ;
	cin >> whatday ;
	
	for (int i = 1; i <= 12; ++i)
	{
		static int days = whatday + 13-1 ;
		days += daysPerMonth[i-1] ;
		if (days % 7 == 5)
			cout << i << endl; 
	}
	 
	return 0 ;
}

File: 2000-2999/OpenJudge2723.java
import java.io.* ;
import java.util.* ;

public class Main {
	public static void main(String[] argv) {
		Scanner cin = new Scanner(System.in) ;
		int days = cin.nextInt() ;
		days += 13-1 ;
		int[] daysPerMonth = {0, 31, 28, 31, 30, 31, 30, 
                31, 31, 30, 31, 30, 12} ;
		for (int i = 1; i <= 12; ++i) {
			days += daysPerMonth[i-1] ;
			if (days % 7 == 5) {
				System.out.println(i);
			}
		}
	}
}